daemon/tests/happy_path.rs

```diff
@@ -152,6 +154,46 @@ async fn collaboratively_close_an_open_cfd() {
     assert_next_state!(CfdState::Closed, maker, taker, order_id);
 }
 
+#[tokio::test]
+#[cfg_attr(not(feature = "fake_clock"), ignore)]
+async fn force_close_an_open_cfd() {
+    let _guard = init_tracing();
+
+    let start_time = OliviaData::example_0().announcement().expected_outcome_time
+        - daemon::SETTLEMENT_INTERVAL
+        - time::Duration::minutes(20);
+
+    let clock_handle = daemon::clock::set_fake_clock(start_time.into());
+
+    let (mut maker, mut taker, order_id) = start_from_open_cfd_state().await;
+
+    std::mem::drop(clock_handle); // No need to fake time anymore
```

Comment:
This would do the dropping for you :)

```suggestion
    let (mut maker, mut taker, order_id) = {
    	let _clock_handle = daemon::clock::set_fake_clock(start_time.into());
    	start_from_open_cfd_state().await
    };
```